PWGEM/PhotonMeson/Tasks/CheckMCV0.cxx

```diff
@@ -9,35 +9,15 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
-/// \brief write relevant information for photon conversion analysis to an AO2D.root file.
-/// dependencies: o2-analysis-lf-lambdakzeromcfinder
```

Comment:
the brief description and dependencies should  stay I think. I thought clang would even complain if they are gone